apmpackage/apm/changelog.yml

```
@@ -3,6 +3,9 @@
     - description: Placeholder
       type: enhancement
       link: https://github.com/elastic/apm-server/pull/123
+    - description: Migrate to package-spec v3
+      type: enhancement
+      link: https://github.com/elastic/apm-server/pull/11822
```

I _think_ automation likes it without Placeholder when cutting a new release, or maybe it works either way
```suggestion
    - description: Migrate to package-spec v3
      type: enhancement
      link: https://github.com/elastic/apm-server/pull/11822
```